CIP-0037/README.md

```diff
@@ -0,0 +1,153 @@
+---
+CIP: CIP-0037 
+Title: Dynamic Saturation 
+Authors: Casey Gibson <caseygibson@protonmail.ch>
+Status: In Review
```

Comment:
```suggestion
Status: Proposed
```
... `Status:` must be one of these keywords 🤓 https://github.com/cardano-foundation/CIPs/tree/master/CIP-0001#cip-statuses
... and as per https://github.com/cardano-foundation/CIPs/pull/163#issuecomment-1136204287 we will merge this as `Status: Proposed` while awaiting the review suggested in your _Path To Active_.